### PR TITLE
Publish via Github Actions

### LINKED ISSUE #175: Migrate CI to Gihub actions
Align this repository with the rest of the ODC ecosystem by using GitHub actions instead of Travis for CI.

### PR DESCRIPTION
**Pull request by whatnick**
Closes #175

### CODE REVIEW

**Review by whatnick — COMMENTED** [review]
We should keep mergify.

**Review by whatnick — COMMENTED** [review]
In a separate future PR remove restcube and other deprecated apps.

**Review by mmochan — APPROVED** [review]
Looks ok to me!

File: .mergify.yml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-pull_request_rules:
```
Comment (NikitaGandhi) [inline .mergify.yml:1]: @whatnick current mergify is specific to travis-ci rule. Need to see an example for GHA

**Pull request merged**